ironfish-cli/src/commands/wallet/transaction.ts

```diff
@@ -45,6 +45,7 @@ export class TransactionCommand extends IronfishCommand {
     this.log(`Transaction: ${hash}`)
     this.log(`Account: ${response.content.account}`)
     this.log(`Status: ${response.content.transaction.status}`)
+    this.log(`Timestamp: ${new Date(response.content.transaction.timestamp).toLocaleString()}`)
```

Comment:
How hard would it be to add a timezone? It's been a minute since I played with JS dates, but that could be nice to clarify "is this UTC or my timezone?"

Reply:
Good point! I'll see what I can do about adding the timezone

Reply:
I like mat's suggestion, but let's do a sweep in another PR for this.

```
$ new Date().toLocaleString('en-us',{timeZoneName:'short'})
'1/17/2023, 1:12:47 PM EST'
```